app/assets/stylesheets/components/header.scss

```diff
@@ -135,8 +135,11 @@
   transform: translate(-50%, -200%);
   transition: transform var(--transition-props);
 
-  &:focus,
-  &:focus-within {
+  &:focus:not(:focus-visible) {
+    visibility: hidden;
+  }
+
+  &:focus-visible {
```

Comment:
small change (blocking) - This is working great on most browsers but Safari still doesn't support `focus-visible` 🙁 - but happy news is that we have a polyfill package called [focus-visible](https://www.npmjs.com/package/focus-visible) that we can use instead. It uses additional classnames to track where focus-visible should be applied - the CSS would need to look something like:

```css
.js-focus-visible .skip-content-link:focus:not(.focus-visible) {
  visibility: hidden;
}

.js-focus-visible .skip-content-link:focus {
  pointer-events: auto;
  transform: translate(-50%, 0);
  outline: none;
  border: 2px solid var(--header-button-focus-color);
}

.skip-content-link {
  position: absolute;
  left: 50%;
  background: var(--body-color);
  color: var(--body-color-inverted);
  z-index: var(--z-popover);
  padding: var(--su-4);
  pointer-events: none;
  border-radius: var(--radius);
  height: var(--header-height);
  transform: translate(-50%, -200%);
  transition: transform var(--transition-props);
}
```

Reply:
My bad, I somehow forgot to test this on Safari... 🤦‍♀️
Will update this, thanks for your help!

Reply:
no worries! it looks we have some other places in the code where we're not using js-focus-visible (I'm going to check if there's any ill effects there 😅), so it's super easy to miss!